### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode character

### ISSUE THREAD

**Issue by steve-heslouin** [Netflix/security_monkey#670]
From the logs 

```
tail -f /var/log/security_monkey/securitymonkey.log
```

```
Traceback (most recent call last):
  File "build/bdist.linux-x86_64/egg/apscheduler/scheduler.py", line 512, in _run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/usr/local/src/security_monkey/security_monkey/scheduler.py", line 32, in run_change_reporter
    reporter.run(account, interval)
  File "/usr/local/src/security_monkey/security_monkey/reporter.py", line 87, in run
    self.account_alerter.report()
  File "/usr/local/src/security_monkey/security_monkey/alerter.py", line 94, in report
    body = report_content(content)
  File "/usr/local/src/security_monkey/security_monkey/alerter.py", line 43, in report_content
    body = template.render(content)
  File "/usr/local/lib/python2.7/dist-packages/Jinja2-2.9.5-py2.7.egg/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/local/lib/python2.7/dist-packages/Jinja2-2.9.5-py2.7.egg/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/src/security_monkey/security_monkey/templates/jinja_change_email.html", line 59, in top-level template code
    {{ new_item.description() }}
  File "/usr/local/src/security_monkey/security_monkey/watcher.py", line 594, in description
    body = template.render(self._dict_for_template())
  File "/usr/local/src/security_monkey/security_monkey/watcher.py", line 584, in _dict_for_template
    'pdiff_html': self.get_pdiff_html()
  File "/usr/local/src/security_monkey/security_monkey/watcher.py", line 574, in get_pdiff_html
    return pdiff.produceDiffHTML()
  File "/usr/local/src/security_monkey/security_monkey/common/PolicyDiff.py", line 420, in produceDiffHTML
    inner_html = diff_dict(self._new_policy, self._old_policy, 1)
  File "/usr/local/src/security_monkey/security_monkey/common/PolicyDiff.py", line 161, in diff_dict
    retstr += added("{4}\"{0}\": {2}<br/>\n{1}{4}{3},".format(keya, print_item(dicta[keya], 'added', indentation + 1), brackets['open'], brackets['close'], i(indentation)))
  File "/usr/local/src/security_monkey/security_monkey/common/PolicyDiff.py", line 142, in print_item
    return print_dict(value, action, indentlevel)
  File "/usr/local/src/security_monkey/security_monkey/common/PolicyDiff.py", line 124, in print_dict
    i(indentation)
UnicodeEncodeError: 'ascii' codec can't encode character u'\u0142' in position 5: ordinal not in range(128)
```

I have a bunch of Polish developer and sounds like they do like to use their own language, anything we can do about it ?

Thanks a lot

**Comment by monkeysecurity** [comment]
I think I have a fix -- but to test it I'm trying to create an AWS primative (security group or iam role or something) with `ł` in the name or description or policy.  AWS won't let me create things that aren't ASCII.

Do you happen to know what technology this is character is being used in?

**Comment by monkeysecurity** [comment]
Ahh, An S3 bucket policy allows non-ASCII characters.

My test passed.

**Comment by monkeysecurity** [comment]
I'm going to merge in my fix to `PolicyDiff.py`.  Could you see if that works for you?

**Comment by monkeysecurity** [comment]
(My fix is fully described in #671.)

**Comment by steve-heslouin** [comment]
Sorry, i couldnt check it earlier, european time was sleeping time :) 
Thanks a lot for your reactivity , i will pull and check it out ;)
